tasks/azure/vm.py

```diff
@@ -90,7 +118,7 @@ def destroy_vm(
     ctx: Context,
     config_path: Optional[str] = None,
     stack_name: Optional[str] = None,
-    yes: Optional[bool] = False,
+    yes: Optional[bool] = True,
```

Comment:
We should remove that. Otherwise you need `--no-yes` to use that option which is weird